creation/server/creation.lua

```diff
@@ -1,115 +1,63 @@
+local parse = {
```

Comment:
These don't need to be in a table. They were fine as freestanding functions. Also, please don't remove the logic to parse them both ways (depending on the `ConfigFormatEnabled` field).

Reply:
Thank you!